cli/smoke.go

```diff
@@ -32,19 +34,32 @@ const (
 
 	// urlPrefix is the localhost prefix for accessing local Evergreen.
 	urlPrefix = "http://localhost:"
+
+	userHeader = "Auth-Username"
+	keyHeader  = "Api-Key"
+
+	hostId     = "localhost"
+	hostSecret = "de249183582947721fdfb2ea1796574b"
```

Comment:
why can't this just be util.RandomString()

Reply:
It has to match the string in the hosts.json. If we start loading the data in scripts/smoke_test/*.json in go instead of in bash the self-tests.yml, we could change this. I opened EVG-2423 for this.